app/models/concerns/patient_searchable.rb

```diff
@@ -1,5 +1,5 @@
 # Methods pertaining to patient search
-module Searchable
+module PatientSearchable
```

Comment:
Looks like there mightbe something to sort out here still -- `search_users` looks like it's still in there?